pkg/registry/service/rest_test.go

```diff
@@ -689,7 +689,7 @@ func TestServiceRegistryIPUpdate(t *testing.T) {
 
 	update = deepCloneService(created_service)
 	update.Spec.Ports[0].Port = 6503
-	update.Spec.ClusterIP = "1.2.3.76" // error
+	update.Spec.ClusterIP = "1.2.4.76" // Error: Provided Cluster IP is not in the range of 1.2.3.0/24
```

Comment:
Thanks a lot for debugging - that's actually what I expected after looking into code.

I agree it's not good if test can randomly fail. I think a solution might be to:
instead of hardcoding "1.2.3.76" value, try to generate "1.2.3.X" as long as it is different than the previous one.
Does it make sense?

It would be great if you could do that.